packages/blade/src/components/Radio/_decisions/decisions.md

```diff
@@ -0,0 +1,109 @@
+# Radio Decisions <!-- omit in toc -->
+
+Radio is used to present each item in a list of options where users must make a single selection.
+
+This document outlines the API of `Radio` & `RadioGroup` component.
+
+<img src="./radio_thumbnail.png" alt="Radio component thumbnail" width="380" />
+
+- [Design](#design)
+- [`Radio` API](#radio-api)
+- [`RadioGroup` API](#radiogroup-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+    - [Controlled, Uncontrolled](#controlled-uncontrolled)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+- [Figma Link - Radio](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13133%3A160709)
+
+## `Radio`
+
+> Note: Radio component can't be used outside of RadioGroup.
+
+| Prop                | Type                                    | Default     | Description                                                                                                                                                    | Required |
+| ------------------- | --------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| children            | `string`                                | `undefined` | The text to be rendered as RadioLabel                                                                                                                          | ✅       |
```

Comment:
What is RadioLabel? Seems like a component which isn't mentioned here

Reply:
Internal component accidentally mentioned here. Will remove.